app/controllers/providers/application_merits_task/nature_of_urgency_controller.rb

```diff
@@ -0,0 +1,27 @@
+module Providers
+  module ApplicationMeritsTask
+    class NatureOfUrgencyController < ProviderBaseController
```

Comment:
I've mentioned this (i.e pluralisation) a couple times across various PRs.

I definitely think it's worthwhile defaulting to convention (e.g NatureOfUrgencies) unless there is a clear reason not to.

How do the rest of the team feel about this?

Reply:
I'd even suggest this could just be an Urgencies controller!

Reply:
i had started with a plural controller name but changed to singular as the route and relations are all singular and convention only [favours pluralisation](https://guides.rubyonrails.org/action_controller_overview.html#controller-naming-convention) but does not adhere to it itself (i.e. ApplicationController).

Reply:
I have renamed the controller (+route etc.) urgencies_controller.rb. @colinbruce @jsugarman @cpjmcquillan are we all happy with the new naming?

Reply:
I would have said its fine to make it plural if its already done as it is not something i think we need to sweat over too much, but `urgencies` for `nature_of_urgenc[ies|y]` i have to say I am not keen on

Reply:
I meant to delete my `urgencies` comment 😆 

I think `NatureOfUrgenciesController` makes most sense in the context of using a `NatureOfUrgency` form model to create `Urgency` records.

Reply:
👍 ok, I have named the controller NatureOfUrgenciesController